### PR TITLE
Milestone # 1 - List Structure

### PR DESCRIPTION
**Pull request by jonathastavares**
Pull request for milestone # 1 

I've already finished this project last week, It's fully working with all functionalities, a pull request was created for project review.

Built with: 
- HTML
- CSS
- Bootstrap
- Javascript
- Webpack

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <body>
       <div id='list-container' class='container w-25 shadow my-5'>
         <ul class="list-group join-list-bottom no-round-borders">
-          <li class="list-group-item font-bold">Today's To Do</li>
+          <li class="list-group-item font-bold">Today's To Do List</li>
           <li class='list-group-item d-flex'><input id='add-list' placeholder='Add to your list...'><button id='add-btn' type='submit' class='btn btn-secondary'>Add</button></li>
         </ul>
         <ul id='list-objects' class="list-group join-list-top join-list-bottom">
```
Comment (Emmyn5600) [inline src/index.html:15]: #### It seems like you are either repeating a week or the code you used 7 days ago, which is complicated to add inline comment however there are issuea you need to implement:

- [x] I would recommend to modify the implementation you are trying to use while initializing as well as populate data.
 
- [x] An array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number].

- [x] It has a function that iterates over the tasks array and populate HTML list item element for each task.

- [x] Make sure data are being stored and retrieved from localStorage

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @jonathastavares 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand: 
    
     - Please check the comment under the review too

    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by jonathastavares** [comment]
All changes are corrected in new-files branch, I deleted other branches to not misunderstood.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         

## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding!clap computer wine_glass
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rloterh — APPROVED** [review]
Checked

**Review by mke2111 — APPROVED** [review]
### Hi @jonathastavares, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 

OPTIONAL
> Ensure that nameless todos can not be created by simply clicking Add button.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**